### PR TITLE
Task/sql error persistence

### PR DESCRIPTION
**Pull request by IvanHdzC**
Solves #1791

This PR inserts all errors on sql persistance on a new table on the DB. If the table doesn't exists, then it's created. The layout for the table is the following.

postgresql=# select * from prueba2.prueba2_error_log;
        **timestamp        |                                error                                |                  query**                                    

 2020-02-27 09:12:42.049 ### **|** ERROR: relation "prueba2.e_farolahttpul_streetlight" does not exist+### **|** INSERT INTO prueba2.e_farola (recvT
ime,fiwareServicePath,entityId,entityType,TimeInstant,TimeInstant_md,serialNumber,serialNumber_md) VALUES ('2020-02-27 15:15:29.344','/e','farolahttpu
l','Streetlight','2019-10-31T13:49:17.00Z','[]','OP00096F160318C000004','[]'),('2020-02-27 15:15:30.510','/e','farolahttpul','Streetlight','2019-10-31
T13:49:17.00Z','[]','OP00096F160318C000004','[]')

for both SQL kind of sinlks.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [cygnus-ngsi][PostgisSink, PostGRESQLSink] Enable datamodel by entity type (dm-by-entity-type) (#1684)
 [cygnus-ngsi] Add correlatorId rollbacked transactions (#1770)
-[cygnus-ngsi][PostGRESQLSink] Fix schemaName when inserting data into DB
\ No newline at end of file
+[cygnus-ngsi][PostGRESQLSink] Fix schemaName when inserting data into DB
+[cygnus-common][PostgreSQL, MySQL] SQL error persistance.
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]:
```suggestion
[cygnus-common][PostgreSQL, MySQL] SQL error persistance (#1791)
```

**Comment by fgalan** [comment]
Documentation should be modified. At least the new table should be described and under which circumstances new records are inserted on it.

**Comment by IvanHdzC** [comment]
> Documentation should be modified. At least the new table should be described and under which circumstances new records are inserted on it.

Fixed in [1e46641](https://github.com/telefonicaid/fiware-cygnus/pull/1823/commits/1e46641b6ac1597718f7a1c1c85e4774de67d2fb)

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (fgalan) [reply CHANGES_NEXT_RELEASE]: Fixed in 9e33016

**Pull request merged**